keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -680,12 +680,10 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
   Knowing etcd configuration will be used to automatically disable the streaming feature.
 
 #### GA
-- Consider using WatchProgressRequester to request progress notifications directly from etcd.
```

Comment:
moved to `beta`

Reply:
can we add that more explicitly to beta?

the detection of etcd isn't the same thing - we should have both

Reply:
ok, done, ptal.